src/comms/config_layer.cpp

```diff
@@ -1,5 +1,17 @@
 #include "config_layer.hpp"
 
+/// @brief This resets the whole processor and kicks it back to program entry (teensy4/startup.c)
+/// @param void void
```

Comment:
Haven't seen this before, but worth documenting since that is just weird historical C that isn't present in C++.
```suggestion
/// @param void specify no arguments (needed in C)
```